scripts/importTypescript.js

```diff
@@ -124,6 +124,7 @@ function importLibs() {
 	};
 
 	enqueue('');
+	enqueue('es6');
 	enqueue('es2015');
```

Comment:
Adding the ES6 means that we can use the TS lib routing pattern 1-to-1, it adds 3-4 LOC (and then the copyright banner)